### ISSUE TITLE
Widevine CDM is missing

### ISSUE THREAD

**Issue by Zero3K** [weolar/xpchrome#10]
*subject*

Please add it back.

**Comment by win32ss** [comment]
Widevine CDM is a separate binary provided by Google. It is completely proprietary and obfuscated, and needs Windows 7 at this time.

**Comment by Alex313031** [comment]
@win32ss @Zero3K Yeah. Widevine uses verified media platform, which relies on media foundation APIs that are only available in Win7 and above. Even the official Chrome M49 (which comes bundled with Widevine and Flash) no longer works for watching Netflix, etc.

Restoring Widevine on XP is probably never going to happen, sadly. But such is the pitfalls of DRM. Another good example is steam ending support for XP, and now 7. (Although at least with 7 it still runs, just doesn't get any updates, versus when they ended XP support and issued one last update that bricked steam from launching at all, and was a big fuck you to anyone with games they paid for, especially ones that only run on XP.)

**Comment by jonm58** [comment]
![ScreenShot_20240128155209](https://github.com/weolar/xpchrome/assets/73036321/88a35adc-0a55-46f8-9646-0e721022a504)
this?
